src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/package.scala

```diff
@@ -14,37 +17,43 @@ package object config {
   }
 
   implicit val dataprocConfigReader: ValueReader[DataprocConfig] = ValueReader.relative { config =>
-    DataprocConfig(config.getString("applicationName"),
-      config.getString("serviceAccount"),
+    DataprocConfig(
+      config.getString("applicationName"),
+      GoogleServiceAccount(config.getString("serviceAccount")),
       config.getString("dataprocDefaultRegion"),
-      config.getString("leoGoogleBucket"),
+      GoogleProject(config.getString("leoGoogleBucket")),
       config.getString("dataprocDockerImage"),
       config.getString("jupyterProxyDockerImage"),
-      config.getString("clusterFirewallRuleName"),
-      config.getString("clusterFirewallVPCNetwork"),
-      config.getString("clusterNetworkTag"),
-      config.getString("configFolderPath"),
-      config.getString("initActionsFileName"),
-      config.getString("clusterDockerComposeName"),
-      config.getString("leonardoServicePemName"),
-      config.getString("jupyterServerCrtName"),
-      config.getString("jupyterServerKeyName"),
-      config.getString("jupyterRootCaPemName"),
-      config.getString("jupyterRootCaKeyName"),
-      config.getString("proxySiteConf"),
       config.getString("clusterUrlBase"),
       config.getString("jupyterServerName"),
-      config.getString("proxyServerName"),
-      config.getString("jupyterInstallExtensionScript"),
-      config.getString("userServiceAccountCredentials"))
+      config.getString("proxyServerName"))
+  }
+
+  implicit val clusterResourcesConfigReader: ValueReader[ClusterResourcesConfig] = ValueReader.relative { config =>
+    val filePrefix = config.getString("configFolderPath")
```

Comment:
what's a config folder path?

Reply:
It's the directory where all of the files in the `ClusterResourcesConfig` exist

Reply:
It won't be needed once we do https://broadinstitute.atlassian.net/browse/GAWB-2625